### PR TITLE
ACM-3663: Fix breadcrumbs

### PR DESCRIPTION
**Pull request by chenz4027**
https://issues.redhat.com/browse/ACM-3663

### CODE REVIEW

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -975,6 +976,8 @@
   "creation.valid.numeric": "Value must be integer.",
   "Credential name": "Credential name",
   "Credential name and namespace are predefined as below for HyperShift add-on": "Credential name and namespace are predefined as below for HyperShift add-on",
+  "Credential type": "Credential type",
+  "Credential Type": "Credential Type",
```
Comment (KevinFCormier) [inline frontend/public/locales/en/translation.json:980]: Can you please use only "Credential type"? Otherwise looks good.

Comment (chenz4027) [reply frontend/public/locales/en/translation.json:980]: Thanks Kevin! Will make the change

Comment (chenz4027) [reply frontend/public/locales/en/translation.json]: Updated :)

**Pull request merged**